docs/sources/release-notes/v2.5.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Grafana Mimir version 2.5 release notes"
+menuTitle: "V2.5 release notes"
+description: "Release notes for Grafana Mimir version 2.5"
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- Alertmanager discord support
+- TLS minimum version and cipher suites are now configurable for HTTP and gRPC servers
+- Various improvements to lower memory usage of store-gateway and ingester
+- New dashboards: "Mimir / Overview resources" and Mimir / Overview networking"
+- Support for read-write deployment mode in jsonnet, this allows users to run Mimir as only three types of services which are responsible for read, write and backend operations
```

Comment:
This is experimental for now. I would not mention in the release notes. Also, we're not used to mention the jsonnet changes in the Mimir release notes.

Reply:
Thx, removed https://github.com/grafana/mimir/pull/3580/commits/504a3c573e2091ddf60941d6615c66cbe05dc183